src/NUnitEngine/nunit.engine/Services/ServiceManager.cs

```diff
@@ -70,23 +69,31 @@ public IService GetService( Type serviceType )
 
         public void AddService(IService service)
         {
-            services.Add(service);
+            _services.Add(service);
             log.Debug("Added " + service.GetType().Name);
         }
 
         public void InitializeServices()
```

Comment:
Since you switched all services to use `StartService` and `StopService` instead of `InitializeService` and `UnloadService`, should this and the next method be renamed to `InitializeServices` and `StopServices` to be consistent?